modules/gapi/include/opencv2/gapi/gcompiled.hpp

```diff
@@ -208,6 +208,19 @@ class GAPI_EXPORTS GCompiled
     // FIXME: Why it requires compile args?
     void reshape(const GMetaArgs& inMetas, const GCompileArgs& args);
 
+    /**
+     * @brief Prepare inner kernels states for a new video-stream.
+     *
+     * GCompiled objects may be used to process video streams frame by frame.
+     * In this case, a GCompiled is called on every image frame individually.
+     * Starting OpenCV 4.4, some kernels in the graph may have their internal
+     * states (see GAPI_OCV_KERNEL_ST for the OpenCV backend).
+     * In this case, if user starts processing another video stream with this
+     * GCompiled, method above needs to be called to let kernels re-initialize
```

Comment:
"Method above" is reshape here. Why not just "this method"?

Reply:
Rewritten to "this method". What is about "re-initialize states to new stream" - should here be "for" instead of "to" or it doesn't matter actually here?